cms/static/js/views/paged_container.js

```diff
@@ -27,7 +26,33 @@ define(["jquery", "underscore", "js/views/utils/view_utils", "js/views/container
                     // of paginator, on the current page.
                     size: function() { return self.collection._size; },
                     // Toggles the functionality for showing and hiding child previews.
-                    showChildrenPreviews: true
+                    showChildrenPreviews: true,
+
+                    // PagingFooter expects to be able to control paging through the collection instead of the view,
```

Comment:
I don't think that refactoring is feasible. From the comment above:
> XBlocks are rendered via Django views and templates rather than underscore templates, and so don't have a Backbone model for us to manipulate in a backbone collection. Here, we emulate the interface of backbone.paginator so that we can use the Paging Header and Footer with this page. As a consequence, however, we have to manipulate its members manually.